index.html

```diff
@@ -1,19 +1,96 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+
+  <body>
+    <header>
+      <img src="img/karma-logo.svg" alt="Karma Logo" />
+      <nav>
+        <ul>
+          <li>
+            <a href="#"><b>Meet Karma</b></a>
+          </li>
+          <li>
+            <a href="#">How it Works</a>
+          </li>
+          <li>
+            <a href="#">Store</a>
+          </li>
+          <li>
+            <a href="#">Blog</a>
+          </li>
+          <li>
+            <a href="#">Help</a>
+          </li>
+          <li>
+            <a href="#">Login</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="intro">
+        <h1>Introducing Karma</h1>
+        <p>Bring WiFi with you, everywhere you go.</p>
+        <button>Learn More</button>
+      </section>
+      <section>
+        <h2>Everyone needs a little Karma</h2>
+        <div class="karma_icon_style">
+          <div class="karma_icon_direction">
+            <img class="karma_icon_images" src="img/icon-devices.svg" />
+            <p>Internet for all devices</p>
+          </div>
+          <div class="karma_icon_direction">
+            <img class="karma_icon_images" src="img/icon-coffee.svg" />
```

Comment:
You should try to have alt text on all your images, to help those reading using a screen reader.

Reply:
Added alt tags to images.